app/components/SearchPopover.tsx

```diff
@@ -126,8 +149,17 @@ function SearchPopover({ shareId }: Props) {
     [popover, searchResults]
   );
 
+  const handleSearchItemClick = React.useCallback(() => {
+    popover.hide();
+    if (searchInputRef.current) {
+      searchInputRef.current.value = "";
+      focusRef.current = document.getElementById(bodyContentId);
+    }
+  }, [popover.hide]);
+
   return (
     <>
+      <RegisterKeyDown trigger="/" handler={handleSearchShortcut} />
```

Comment:
Prefering `useKeyDown` btw, this component is just for use in the old class components where we can't use hooks